src/ocm/components/clusters/ClusterPageErrors.tsx

```diff
@@ -2,13 +2,9 @@ import React from 'react';
 import { Button, ButtonVariant, PageSection, PageSectionVariants } from '@patternfly/react-core';
 import { Link } from 'react-router-dom';
 import { ErrorState } from '../../../common';
-import { isSingleClusterMode, routeBasePath } from '../../config';
+import { routeBasePath } from '../../config';
 
 export const getErrorStateActions = () => {
```

Comment:
Similarly to the other case, now  `getErrorStateActions` can be a constant list, it doesn't need to be a function anymore. (and possibly we could extract it to a single place in case they are the same)

Reply:
Addressed in bfac0e1b807c523bfa47cb2c1f3803deec4b8623